diff --git a/src/3rdparty/chromium/third_party/blink/renderer/core/streams/underlying_sink_base.h b/src/3rdparty/chromium/third_party/blink/renderer/core/streams/underlying_sink_base.h
index 0d904ee9778..b9bef8a032a 100644
--- a/src/3rdparty/chromium/third_party/blink/renderer/core/streams/underlying_sink_base.h
+++ b/src/3rdparty/chromium/third_party/blink/renderer/core/streams/underlying_sink_base.h
@@ -8,6 +8,7 @@
 #include "third_party/blink/renderer/bindings/core/v8/script_promise.h"
 #include "third_party/blink/renderer/bindings/core/v8/script_value.h"
 #include "third_party/blink/renderer/core/core_export.h"
+#include "third_party/blink/renderer/core/streams/stream_algorithms.h"
 #include "third_party/blink/renderer/platform/bindings/script_wrappable.h"
 #include "third_party/blink/renderer/platform/heap/visitor.h"
 
